app.py

```diff
@@ -13,70 +15,136 @@
 import src.persistent_state
 import src.settings
 import src.swap_amm
-import src.utils
-from src.chart_utils import (
-    get_protocol_data_mappings,
-    load_stats_data,
-    transform_loans_data,
-    transform_main_chart_data,
-)
 
-PROTOCOL_NAMES = [
```

Comment:
This should be removed

Reply:
hello @djeck1432 this requested change is not in my file, but it is currently in your file. Or do you mean i should add it to my file?

> This should be removed

Reply:
But in your code, you removed `PROTOCOL_NAMES`